### PR TITLE
Update user guide

### PR DESCRIPTION
**Pull request by underthehai**
Update Sort Command section to sort date, datetime, mobile, and passport

Add tutorial guide section for a tutorial guide to use the app
Add clear command

### CODE REVIEW

**Review by jiaweiteo — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -52,35 +53,57 @@ It is optimized for CLI users so that destinations and details can be added fast
...
+## Tutorial Guide
+
+Here are the steps you can follow to get familiar with _Wanderlust_
+
+Command | Description
+------------ | -------------
+`clear` | Remove all preset data from Wanderlust
+`add -travelplan n/Singapore sd/2021-05-05 ed/2021-06-06` | Adds a Singapore travelPlan to the travelPlan list 
+`goto -travelplan 1` | Go to the Singapore travelplan
+`add -friend n/Tom p/S1234567 m/12345678` | Add a friend name `Tom` with passport number `S1234567` and mobile number `12345678`
```
Comment (lyeyixian) [inline docs/UserGuide.md:95]: The passport and mobile field needs to be changed, to reflect the new Regex. 
Passport: [STFG] + 7 numbers + [A-Z]
Mobile: [89] + 7 numbers

File: docs/UserGuide.md [hunk 2]
```diff
@@ -117,25 +141,29 @@ Wishlist | TravelPlan
...
-`c/cost` | Cost of the activity, if any. | Cost should only contain numbers, and it should be a positive integer.
-`d/DATE_AND_TIME` | Date and Time intended to do the activity | Format of date is in YYYY-MM-DD and format of time is HH:MM (24h clock).
+`l/LOCATION` | Location/ Address of the activity | Location can take any values, and it should not be blank.
+`i/LEVEL_OF_IMPORTANCE` | The priority assigned to the activity | Importance Level should only contain numbers, and it should range from 1 - 5, with 5 being the most important while 1 being the least important.
+`c/COST` | Cost of the activity | Cost should only contain numbers, and it should be a positive integer.
+`d/DATE_AND_TIME` | Date and Time intended to do the activity | Format of date is in YYYY-MM-DD and format of time is HH:MM (24h clock). 
+
+**Notes about Activity:**<br>
+
+* Different Activity can have the same datetime within _Wanderlust_ travelplanner
```
Comment (lyeyixian) [inline docs/UserGuide.md:158]: Is it within travel plan instead?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -147,6 +175,10 @@ Name of Parameter | Description | Requirement
 `m/MOBILE_NUMBER` | Mobile number of the friend cell mobile | Must be 8 digits only.
```
Comment (lyeyixian) [inline docs/UserGuide.md:175]: Mobile and Passport parameters need update

File: docs/UserGuide.md [hunk 4]
```diff
@@ -147,6 +175,10 @@ Name of Parameter | Description | Requirement
...
 `p/PASSPORT_NUMBER` | Passport number of the friend passport | Passport numbers should only contain 1 character and 7 numbers (To be updated with standard passport conventions in the future).
 
+**Notes about Friend:**<br>
+
+* Passport is case-insensitive
```
Comment (lyeyixian) [inline docs/UserGuide.md:180]: Actually now it's case-sensitive XD

File: docs/UserGuide.md [hunk 5]
```diff
@@ -227,8 +269,10 @@ Example: `add -travelplan n/France sd/2020-09-15 ed/2020-09-30`
 
 
 ### 2. Adding an Activity (L)
-Creates a new activity and adds it to the travel plan/wishlist in the current directory. Date and time can be optional, but they must exist as a pair.
-Format of date is in YYYY-MM-DD and format of time is HH:MM (24h clock).
+Creates a new activity and adds it to the travel plan/wishlist in the current directory.
+Format of date is in YYYY-MM-DD and format of time is HH:MM (24h clock).<br/>
+Activities can have the same datetime within the travelplanner.
```
Comment (lyeyixian) [inline docs/UserGuide.md:274]: Same as above.

**Review by lyeyixian — APPROVED** [review]
LGTM. Mobile and Passport need updates

**Pull request merged**